### PR TITLE
Add broker_count and topic_partition_under_replicated metric

### PR DESCRIPTION
**Pull request by smlgbl**
For us the most important metric next to the consumer lags, in case a broker goes down or there's a failure in replication.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,10 @@ Kubernetes users may want to use the Helm chart to deploy Kafka Minion: https://
 | KAFKA_TLS_CERT_FILE_PATH | Path to the TLS cert file | (No default) |
 | KAFKA_TLS_INSECURE_SKIP_TLS_VERIFY | If true, TLS accepts any certificate presented by the server and any host name in that certificate. | true |
 | KAFKA_TLS_PASSPHRASE | Passphrase to decrypt the TLS Key | (No default) |
+| KAFKA_TLS_PASSPHRASE | Passphrase to decrypt the TLS Key | (No default) |
```
Comment (weeco) [inline README.md]: Duplicate

**Comment by weeco** [comment]
I should be able to review this PR during this weekend. Thanks for submitting.

In general we use the JMX metrics for underreplicated partitions. Is there a reason why you wouldn't use them and have the metrics preferably reported by Kafka Minion? In general Kafka Minion focuses on Consumer Group specific metrics, rather than Broker metrics.

**Comment by smlgbl** [comment]
Thanks for considering our PR.
We weren't able to find a nice configurable jmx-exporter to prometheus, similar to what we have as a graphite exporter now. If you have any hints, that would be appreciated. Otherwise with this PR kafka_minion would provide all the metrics we need so far.

**Comment by andpol** [comment]
@smlgbl, we're using https://github.com/prometheus/jmx_exporter as a Kubernetes sidecar container for several different JVM apps (including Kafka itself) with good results.

**Comment by smlgbl** [comment]
We moved to jmx_exporter and don't need this PR anymore.

**Pull request closed without merging**